community/cloud-foundation/templates/healthcheck/healthcheck.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2018 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This template creates a healthcheck."""
+
+
+def set_if_exists(healthcheck, properties, prop):
+    """
+    If prop exists in properties, set the healthcheck's property to it.
+    Input:  [dict] healthcheck: a dictionary representing a healthcheck object
+            [dict] properties: a dictionary of the user supplied values
+            [string] prop: the value to check if exists within properties
+
+    """
+    if prop in properties:
+        healthcheck[prop] = properties.get(prop)
```

Comment:
Please avoid the .get() syntax, use properties[prop] instead.

Reply:
Done, thanks Adam.